src/frontend/context/ConfigContext.tsx

```diff
@@ -67,19 +67,23 @@ export type State = {
   fields: FieldsMap;
   metadata: Metadata;
   messages: Messages;
-  status: Status;
+  status?: Status;
```

Comment:
nit but probably shouldn't be optional?

Reply:
ah nvm i see you added the annotation to the default state, which doesn't have the field defined